test/asn1_encode_test.c

```diff
@@ -194,6 +198,7 @@ static ASN1_LONG_DATA long_expected_32bit[] = {
     { 0xff, 0, 1 }, { 0, 0, 0 }, /* t_zero */
     { 0, 0, 0 }, { 0xff, 1, 0x7fffffff }, /* t_longundef */
     CUSTOM_EXPECTED_SUCCESS(1, 1), /* t_one */
+    CUSTOM_EXPECTED_SUCCESS(-1, -1), /* t_one_neg */
```

Comment:
Was it actually impossible to arrange direct reference to corresponding vector? I mean as opposite to chasing indexes through cross-referencing comments?